week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/model/CartPanel.d.ts

```diff
@@ -0,0 +1,7 @@
+import { Item } from './Item'
+
+export interface CartPanel {
+    heading?: string;
+    items: Array<Item>;
+}
+
```

Comment:
to jest "prop-type" dla `cart-panel.component` i może mieszkać razem z `cart-panel.component`

Reply:
Zrobione.